kDrive/UI/Controller/Menu/SharedWithMe/SharedWithMeViewModel.swift

```diff
@@ -20,6 +20,69 @@ import kDriveCore
 import RealmSwift
 import UIKit
 
+/// Public share view model, loading content from memory realm
+final class PublicShareViewModel: InMemoryFileListViewModel {
+    var publicShareProxy: PublicShareProxy?
+    let rootProxy: ProxyFile
+    var publicShareApiFetcher: PublicShareApiFetcher?
+
+    required init(driveFileManager: DriveFileManager, currentDirectory: File? = nil) {
+        guard let currentDirectory else {
+            fatalError("woops")
```

Comment:
Couldn't currentDirectory be not nil instead of fatal on guard ?

Reply:
yup, done later, in the [multi select PR](https://github.com/Infomaniak/ios-kDrive/pull/1311/files#diff-b975f8c73fc66cbdc9388fae38913b7318f087cea0ccb25cffa391db4066035d)